static/app/constants/index.tsx

```diff
@@ -309,6 +309,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Monitor Check-Ins'),
     uid: 10,
   },
+  [DataCategoryExact.MONITOR_SEAT]: {
```

Comment:
derived from https://github.com/getsentry/getsentry/blob/e657f445c1facd71fcee48682cf4e0ed6dd73b24/getsentry/billing/utils/data_category.py#L170, but some of these usages/relationships are a bit unclear lmk if something doesn't make sense here